### DISCUSSION TITLE
iPhone Notch & styling

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#11704] [General - Components / Directives / etc] [answered]
Is there a way to detect iPhone version and style the app accordingly? This applies mainly to the notch on the top which makes some buttons on the top not so easily accessible for the user.

**Comment by MilosPaunovic** [comment, marked as answer]
Check this out:
[https://quasar.dev/quasar-cli/developing-capacitor-apps/troubleshooting-and-tips#status-bar-and-notch-safe-areas](https://quasar.dev/quasar-cli/developing-capacitor-apps/troubleshooting-and-tips#status-bar-and-notch-safe-areas)

**Reply by i-mage** [reply]
Brilliant, thanks! Is there an option for Cordova as well?

**Reply by MilosPaunovic** [reply]
[https://quasar.dev/quasar-cli/developing-cordova-apps/troubleshooting-and-tips#status-bar-and-notch-safe-areas](https://quasar.dev/quasar-cli/developing-cordova-apps/troubleshooting-and-tips#status-bar-and-notch-safe-areas)

**Reply by i-mage** [reply]
Thank you! :-)
